src/main/java/com/qaprosoft/carina/demo/mobile/gui/mfp/pages/android/ConfirmDeletePage.java

```diff
@@ -0,0 +1,25 @@
+package com.qaprosoft.carina.demo.mobile.gui.mfp.pages.android;
+
+import com.qaprosoft.carina.core.foundation.utils.factory.DeviceType;
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.demo.mobile.gui.mfp.pages.common.ConfirmDeletePageBase;
+import com.qaprosoft.carina.demo.mobile.gui.mfp.pages.common.DiaryPageBase;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = ConfirmDeletePageBase.class)
+public class ConfirmDeletePage extends ConfirmDeletePageBase {
```

Comment:
this isn't a standalone page, ConfirmDelete modal is a part of EditDiaryPage, no need to create more page objects that needed

Reply:
So, is it better to move ConfirmDeletePage class inside EditDiaryPage class, or just move methods and locators?